src/arch/src/aarch64/regs.rs

```diff
@@ -29,6 +27,15 @@ pub enum Error {
 }
 type Result<T> = result::Result<T, Error>;
 
+impl fmt::Debug for Error {
```

Comment:
nit: this looks more like a `Display` impl, I think you could leave both the `Debug` derive (for unwraps / tests / ?) and convert this to a `Display`

Reply:
I applied your suggestion, take a look!